pkg/controller/controller.go

```diff
@@ -140,11 +143,20 @@ func (c *Controller) initController() {
 	}
 }
 
-func (c *Controller) runInformers(stopCh <-chan struct{}) {
-	go c.postgresqlInformer.Run(stopCh)
+func (c *Controller) runPodInformer(stopCh <-chan struct{}, wg *sync.WaitGroup) {
+	defer wg.Done()
+	wg.Add(1)
```

Comment:
I'd move all wg.Add-s to the Run. BTW, we don't need wg.Done and wg.Add for the Run itself, there is no way for the execution to reach wg.Wait without exiting Run first.